### DISCUSSION TITLE
A Carla tutorial series in Chinese Zhihu

### DISCUSSION THREAD

**Discussion by xfqbuaa** [carla-simulator/carla#6398] [Show and tell]
A Carla tutorial series in Chinese Zhihu 
https://www.zhihu.com/column/c_1509207053185118208

**Comment by dajiba6** [comment]
this link doesn't work

**Reply by xfqbuaa** [reply]
https://www.zhihu.com/column/c_1509207053185118208

**Reply by xfqbuaa** [reply]
I update the link, please try it again.

**Comment by xfqbuaa** [comment]
Please try it again https://www.zhihu.com/column/c_1509207053185118208
Zhihu is a Chinese website. 
You can also get python file from: https://github.com/xfqbuaa/carla_simulator_Chinese/tree/master/proj

**Reply by dajiba6** [reply]
thank you my friend, i love you

**Comment by yyhtbs-yye** [comment]
<img width="455" alt="image" src="https://github.com/carla-simulator/carla/assets/74354145/f666800c-0e88-4833-a4cc-4ee341aa5fc4">
That is really a nice tutorial, I am current using Carla to capture road traffic, however, the camera image seems very noisy, as shown above. 
Do you know what causes this problem? Or do I need to compile the whole stuff to distable some camera rendering features.

**Comment by WPeilin** [comment]
哥们儿，牛逼

**Comment by PraveenKumar-Rajendran** [comment]
@xfqbuaa Nice work!! Can you please do a similar tutorial in English if you have time? I look forward to your response.
Thanks in advance!

**Comment by LifeIsBinary** [comment]
good job!~
